.azure-pipelines/integration-tests.yml

```diff
@@ -228,7 +228,7 @@ jobs:
   pool:
     vmImage: windows-2019
   variables:
-    msiOutputDirectory: deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/bin/$(buildConfiguration)/$(buildPlatform)/en-us
+    msiOutputDirectory: src/Datadog.Trace.ClrProfiler.WindowsInstaller/bin/$(buildConfiguration)/$(buildPlatform)/en-us
```

Comment:
Feel free to rename `Datadog.Trace.ClrProfiler.WindowsInstaller` (directory and `wixproj` file) to something shorter, like `Datadog.Trace.Msi` or even simply `WindowsInstaller`. Unlike .NET projects, there's no assembly or namespace convention to match, and it doesn't match the artifact name (`datadog-dotnet-apm-{version}-{platform}.msi`), either, so it's kinda arbitrary.

Reply:
Good idea, I'll rename to `WindowsInstaller`